docs/whats-new/staging-release-notes.md

```diff
@@ -22,18 +22,10 @@ For more information, refer to the following:
 <details>
 <summary style="font-size:20px;font-weight:bold">Fixed the issue that triggered incorrect emails from TechBiz</summary>
 
- A fix has been applied to the email templates to correct the invitation emails triggered from TechBiz.
+A fix has been applied to the email templates to correct the invitation emails triggered from TechBiz.
 
 </details>
 
-**Fixes** - **TechPass portal**
-
-<details>
-<summary style="font-size:20px;font-weight:bold">UI updated in Managed User page</summary>
-
-We have updated the **Email ID** field displayed on the **Managed User** page.
-</details>
-
```

Comment:
i'm thinking of leaving this out since neither our users or tenants would know about this page

Reply:
agree